ts_scripts/print_env_info.py

```diff
@@ -134,6 +171,8 @@ def check_release_file():
 def get_os():
     from platform import machine
     platform = get_platform()
+    if platform == 'win32' or platform == 'cygwin':
```

Comment:
We should rather use windows-powershell which is inbuilt, instead of cygwin.

Reply:
sys.platform returns values:

System | platform value
-- | --
AIX | 'aix'
Linux | 'linux'
Windows | 'win32'
Windows/Cygwin | 'cygwin'
macOS | 'darwin'

That's why I put 'cygwin'